pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "brew-control-dash"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
you can either fill this in or i think an empty list is ok if you want to omit it